code/game/objects/structures/stairs.dm

```diff
@@ -11,7 +11,9 @@
 	icon = 'icons/obj/stairs.dmi'
 	icon_state = "stairs"
 	anchored = TRUE
-	resistance_flags = INDESTRUCTIBLE // Skrat Edit : Prevents blobbos from cheesing and destroying irreplacable stairs.
+//SKYRAT EDIT START
+	resistance_flags = INDESTRUCTIBLE // Skrat Edit : Makes OBJ stairs indestructible to primarily prevent blob cheese.
+//SKYRAT EDIT END
```

Comment:
this is still nonmodular-- you'll need to find a place you can put this modularly, or create a dm file for it.